phoenix-core/src/main/java/org/apache/phoenix/compile/DeleteCompiler.java

```diff
@@ -748,6 +757,14 @@ public Long getEstimateInfoTimestamp() throws SQLException {
         }
     }
 
+    /**
+     * Implementation of MutationPlan that is selected if
+     * 1) there are mutable indexes presented for the table,
+     * 2) auto commit is on,
```

Comment:
`auto commit is on` => `auto commit is enabled as well as server side delete mutations are enabled`

thanks @tkhurana